File: test/fsharpTests/BoolTest/compareTest.fs
testCase "compare(true,true)" 
<| fun _ -> 
    let expected = 0
    Expect.equal expected (Bool.compare true true) "error"
testCase "compare(true,false)" 
<| fun _ -> 
    let expected = 1
    Expect.equal expected (Bool.compare true false) "error"
testCase "compare(false,true)" 
<| fun _ -> 
    let expected = -1
    Expect.equal expected (Bool.compare false true) "error"
testCase "compare(false,false)" 
<| fun _ -> 
    let expected = 0
    Expect.equal expected (Bool.compare false false) "error"

File: test/ocamlTests/BoolTest/compareTest.ml
test "compare(true,true)" (fun () -> expect (Bool.compare true true) |> toEqual Eq.int 0) ; 
test "compare(true,false)" (fun () -> expect (Bool.compare true false) |> toEqual Eq.int 1) ; 
test "compare(false,true)" (fun () -> expect (Bool.compare false true) |> toEqual Eq.int -1) ; 
test "compare(false,false)" (fun () -> expect (Bool.compare false false) |> toEqual Eq.int 0) ; 
